source/adios2/operator/compress/CompressPNG.h

```diff
@@ -0,0 +1,87 @@
+/*
+ * Distributed under the OSI-approved Apache License, Version 2.0.  See
+ * accompanying file Copyright.txt for details.
+ *
+ * CompressPNG.h
+ *
+ *  Created on: Jun 10, 2019
+ *      Author: William F Godoy godoywf@ornl.gov
+ */
+
+#ifndef ADIOS2_OPERATOR_COMPRESS_COMPRESSPNG_H_
+#define ADIOS2_OPERATOR_COMPRESS_COMPRESSPNG_H_
+
+#include <set>
+
+#include "adios2/core/Operator.h"
+
+namespace adios2
+{
+namespace core
+{
+namespace compress
+{
+
+class CompressPNG : public Operator
```

Comment:
The thing I worry about with this file is that since png is a lossless format, there's very little scope for high compression ratios. So I worry that users will futz around with this capability without actually generating wins.

Have you seen any wins from playing with the Zlib compress options exposed in `libpng`?

Reply:
The serializer in adios2 is just providing memory space. Generating wins is independent of adios2, it really comes down to the application inputs: data characteristics and settings and what the chosen compressor can handle.